cupy/cuda/compiler.py

```diff
@@ -582,9 +582,12 @@ def _compile_with_cache_hipcc(source, options, arch, cache_dir, extra_source,
                 mod.load(binary)
                 return mod
 
-    # TODO(leofang): catch HIPCCException and convert it to CompileException
-    # with backend='hipcc'
-    binary = _hipcc(source, options, arch)
+    if backend == 'nvrtc':
```

Comment:
1. Ditto for the backend name:
```suggestion
    if backend == 'hiprtc':
```
2. Could you leave a short comment here to explain `compile_using_nvrtc` calls hiprtc for HIP builds?